admin-ui/app/project/info/index.tsx

```diff
@@ -57,7 +57,7 @@ const InfoProjectComponent = ({ items }: any) => {
         <Grid item xs={1}>
           <Grid container>
             <Grid item xs={3.8}>
-              <Link href={"/testing/-1"} style={{ textDecoration: "none" }}>
+              <Link href={"/project/-1"} style={{ textDecoration: "none" }}>
```

Comment:
all url should be constant and no style a with proper Link attribute, https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag